src/test/test-cgroup-setup.c

```diff
@@ -57,6 +58,9 @@ static void test_is_wanted(void) {
 }
 
 int main(void) {
+        if (access("/proc/cmdline", R_OK) == -1 && errno == EACCES)
```

Comment:
This should be below `test_setup_logging()`.

Reply:
we usually check for syscall success with `< 0` rather than `== -1`